features/buyer/award_flow_requirements.feature

```diff
@@ -0,0 +1,82 @@
+@awardflow @skip-staging
+Feature: Award Flow
+  Award a brief to a supplier using the wawrd flow accessible from the buyer dash
```

Comment:
Tiny typo in `wawrd` here (and the staging version)

Reply:
Done